app/schema.py

```diff
@@ -91,7 +91,7 @@ class MenteeUpdate(BaseModel):
     tech_stack: Optional[constr(max_length=255)]
     job_help: Optional[bool]
     pair_programming: Optional[bool]
-    referred_by: Optional[constr(max_length=255)]
+    referred_by: Optional[constr(max_length=255)] 
```

Comment:
what changed here?

Reply:
There was a comment there that I added to remind myself about something.  But, I deleted the comment and made no changes to the code. Is that enough to make it think we made a change?

Reply:
oh, i see. There's a space at the end of the line that wasn't there before, not a huge deal.